interfaces/cython/SConscript

```diff
@@ -47,6 +47,13 @@ localenv.Prepend(CPPPATH=[Dir('#include'), inc, numpy_include])
 localenv.Prepend(LIBS=localenv['cantera_libs'])
 if localenv['OS'] == 'Darwin':
     localenv.Append(LINKFLAGS='-undefined dynamic_lookup')
+    # Don't print deprecation warnings for internal Python changes.
+    # It seems that only clang on macOS prints these deprecation warnings.
+    # Only applies to Python 3.8 and Cython < 3.0. The source of the warnings
+    # will be removed after Python 3.8, so the minimum version of Cython will
+    # need to change for Python 3.9.
+    if LooseVersion(py_version) == LooseVersion("3.8"):
```

Comment:
Given the documentation for `distutils.sysconfig.get_python_version()`, this could just be:
```suggestion
    if py_version == "3.8":
```